src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -55,15 +55,15 @@ public PersonCard(Recipe recipe, int displayedIndex) {
 
         id.setText(displayedIndex + ". ");
         name.setText(recipe.getName().name);
-        phone.setText(recipe.getPhone().value);
-        email.setText(recipe.getEmail().value);
+        ingredients.setText(recipe.getIngredients().ingredientListString);
+        instructions.setText(recipe.getInstructions().instructionListString);
```

Comment:
@AdarshChugani as well as the method called

Reply:
👍  Thanks Kev!